src/components/Accordion/_index.scss

```diff
@@ -8,46 +8,36 @@
 
 @include export-namespace($name: 'accordion') {
   .#{$prefix}--accordion {
-    &__heading {
-      flex-direction: row;
-    }
-
-    &__arrow {
-      margin: 0 0 0 $carbon--spacing-03;
-      transform: rotate(0deg);
-    }
-
-    &__item--active {
-      .#{$prefix}--accordion__title {
-        font-weight: 600;
+    &.#{$namespace}accordion--left {
```

Comment:
- [ ] Can we import `globals/namespace/index` here if we're leveraging this variable? It will be required once we finish the style modularising work - thanks!